packages/core/src/server/index.ts

```diff
@@ -68,9 +70,10 @@ function getFinalAPISchema() {
 export const getEnvelop = async () =>
   envelop({
     plugins: [
+      useEngine(GraphQLJS),
       useSchema(getFinalAPISchema()),
       useExtendContext(apiContextFactory),
-      useMaskedErrors({ formatError }),
+      useMaskedErrors({ maskError: customFormatError }),
```

Comment:
Ref: https://the-guild.dev/graphql/envelop/v3/guides/migrating-from-v2-to-v3#8-update-options-for-usemaskederrors-plugin